config/etc/system-image/writable-paths

```diff
@@ -13,6 +13,7 @@
 /tmp                                    none                    temporary   none        defaults
 /mnt                                    none                    temporary   none        defaults
 /media                                  none                    temporary   none        defaults
+/boot/androidboot                       auto                    persistent  none        none
```

Comment:
well, even when you want to use a config file, this should simply not require a heavy redesign of the current setup ... by adding a writable dir at the place where the system-boot partition gets mounted you also add a requirement to change current processes. if you insist oin having a config, lets just have the file sit in system-boot instead and have it mounted under /boot/<bootloader> by default like all the other bootloader implementations do. i dont see a reason for having any additional writable dir.